docs/userguide/networking/dockernetworks.md

```diff
@@ -0,0 +1,368 @@
+$ sudo docker run -itd --name=container1 busybox
+f2870c98fd504370fb86e59f32cd0753b1ac9b69b7d80566ffc7192a82b3ed27
+
+$ sudo docker run -itd --name=container2 busybox
+bda12f8922785d1f160be70736f26c1e331ab8aaf8ed8d56728508f2e2fd4727
+```
+
+And then inspect the `bridge` network again. You can recognize both newly launched containers by their ids inside the network.
+
+```
+$ sudo docker network inspect bridge
+{
+    "name": "bridge",
+    "id": "7fca4eb8c647e57e9d46c32714271e0c3f8bf8d17d346629e2820547b2d90039",
+    "driver": "bridge",
+    "containers": {
+        "bda12f8922785d1f160be70736f26c1e331ab8aaf8ed8d56728508f2e2fd4727": {
+            "endpoint": "e0ac95934f803d7e36384a2029b8d1eeb56cb88727aa2e8b7edfeebaa6dfd758",
+            "mac_address": "02:42:ac:11:00:03",
+            "ipv4_address": "172.17.0.3/16",
+            "ipv6_address": ""
+        },
+        "f2870c98fd504370fb86e59f32cd0753b1ac9b69b7d80566ffc7192a82b3ed27": {
+            "endpoint": "31de280881d2a774345bbfb1594159ade4ae4024ebfb1320cb74a30225f6a8ae",
+            "mac_address": "02:42:ac:11:00:02",
+            "ipv4_address": "172.17.0.2/16",
+            "ipv6_address": ""
+        }
+    }
+}
+```
+`docker network inspect` command above shows all the connected containers and its network resources on a given network
+
+Containers in this default network are able to communicate with each other using container names.  First `attach` to your running `container` and investigate its  configuration:
+
+```
+$ sudo docker attach container1
+
+/ # ifconfig
+eth0      Link encap:Ethernet  HWaddr 02:42:AC:11:00:02
+          inet addr:172.17.0.2  Bcast:0.0.0.0  Mask:255.255.0.0
+          inet6 addr: fe80::42:acff:fe11:2/64 Scope:Link
+          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
+          RX packets:17 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:3 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:1382 (1.3 KiB)  TX bytes:258 (258.0 B)
+
+lo        Link encap:Local Loopback
+          inet addr:127.0.0.1  Mask:255.0.0.0
+          inet6 addr: ::1/128 Scope:Host
+          UP LOOPBACK RUNNING  MTU:65536  Metric:1
+          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
+          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
+          collisions:0 txqueuelen:0
+          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
+```
+
+Then use `ping` for about 3 seconds to test the connectivity of the containers on this `bridge` network.
+
+```
+/ # ping -w3 container2
+PING container2 (172.17.0.3): 56 data bytes
+64 bytes from 172.17.0.3: seq=0 ttl=64 time=0.125 ms
+64 bytes from 172.17.0.3: seq=1 ttl=64 time=0.130 ms
+64 bytes from 172.17.0.3: seq=2 ttl=64 time=0.172 ms
+
+--- container2 ping statistics ---
+3 packets transmitted, 3 packets received, 0% packet loss
+round-trip min/avg/max = 0.125/0.142/0.172 ms
+```
+
+Finally, use the `cat` command to check the `container1` network configuration:
```

Comment:
Not sure we should refer to `/etc/hosts` as "network configuration", because it's only for resolving IP addresses?

@mavenugo suggestions for another name?